### DISCUSSION TITLE
怎么可以扩展Vant组件？

### DISCUSSION THREAD

**Discussion by jliang87** [youzan/vant#7724] [Q&A]
我想创建基于Vant的自定义组件，例如Contact-Card，现在只有名字和电话能显示，我想扩展到可以展示地址，或者更复杂的东西。我试用extends/mixin，但是发现render function都会完全覆盖Vant里本身的，lifescycle方法例如mounted完全用不了。请求帮助。

**Comment by nbollard4** [comment]
Hello
